mne/io/constants.py

```diff
@@ -912,4 +914,4 @@
 FIFF.FIFFB_MNE_ANNOTATIONS         = 3810  # annotations block
 
 # MNE Metadata Dataframes
-FIFF.FIFFB_MNE_METADATA            = 3811  # metadata dataframes block
+FIFF.FIFFB_MNE_METADATA            = 3811  # metadata dataframes block
```

Comment:
see if you can get this to not show up as a difference , I don't think you changed this line

Reply:
maybe I delete a blank line at the end? I added one back so hopefully it helps

Reply:
once you push you can open up this pull request, look at the files changed tab, and see if it worked yourself